addons/web/static/src/model/relational_model/dynamic_list.js

```diff
@@ -239,6 +239,19 @@ export class DynamicList extends DataPoint {
     // Protected
     // -------------------------------------------------------------------------
 
+    async _copyRecords(resIds) {
```

Comment:
I would inline the function inside _duplicateRecords, because like this it's unclear: there're 2 methods with a similar name, people won't know which one to use.